integration-tests/integration/coordinator/add-involved-staff.spec.js

```diff
@@ -66,7 +66,7 @@ context('A use of force coordinator can add involved staff', () => {
       .statements()
       .then(result =>
         expect(result).to.deep.equal([
-          { username: 'TEST_USER name', link: 'View statement', isOverdue: false, isUnverified: false },
+          { username: 'TEST_USER name', badge: '', link: 'View statement', isOverdue: false, isUnverified: false },
```

Comment:
Introduced 'badges' in integration tests to refer to the OVERDUE, REMOVAL REQUEST and EMAIL NOT VERIFIED components. This differentiates them from the href links.